### DISCUSSION TITLE
World hopper - split USA region filter

### DISCUSSION THREAD

**Discussion by FungiRS** [runelite/runelite#17175] [Ideas]
Split for USA east and west please :)
![RuneLite_I7ic9Sn6Ix](https://github.com/runelite/runelite/assets/78319558/b5296852-727d-4d77-9fc3-c378e45dcebc)

**Comment by Alexgopen** [comment]
Seconding this, I currently filter to USA region, but I want to filter to only US West.  Splitting up the USA region filter to US East and US West would help.
